### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by konevm**
Create router and footer.

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -1,11 +1,18 @@
...
 
 function App() {
   return (
     <div className="app">
-      <Header />
+      <Routes>
+        <Route path="/" element={<Layout />}>
+          <Route index element={<MainPage />} />
+          <Route path="*" element={<MainPage />} />
+        </Route>
```
Comment (bobkov-evgeny) [inline src/App.tsx:14]: Let's add redirect, which will navigate user to "/" route, but not render <MainPage/>

File: src/components/Footer/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import "./Footer.scss";
+
+const Footer: React.FC = () => {
+  const { menu: menuList, payments } = useAppSelector((state) => state.data);
+
+  return (
+    <footer className="app__footer">
+      <div className="footer__menu">
+        {menuList.map((item) => (
+          <Link key={item} to={`/${item}`} className="footer__menu-item">
```
Comment (bobkov-evgeny) [inline src/components/Footer/Footer.tsx:16]: If I'm not mistaken at menuList item will look like "Terms Of Service", but we can't have route path like that

**Review by bobkov-evgeny — APPROVED** [review]

File: src/App.tsx (hunk 1 above)
Comment (konevm) [reply src/App.tsx:14]: done

File: src/components/Footer/Footer.tsx (hunk 2 above)
Comment (konevm) [reply src/components/Footer/Footer.tsx]: done

**Pull request merged**